fuel/transformers/__init__.py

```diff
@@ -21,11 +21,17 @@ class Transformer(AbstractDataStream):
         automatically requested from the wrapped data stream and stored in
         this attribute. Use it to access data from the wrapped data stream
         by calling ``next(self.child_epoch_iterator)``.
+    batch_input : boolean
+        Specification whether the input stream is on example or batch
+
+    batch : boolean
```

Comment:
Not sure how this got reintroduced, but we now have `batch_input` and `batch` in the docstring.